### PR TITLE
Add the fuzz testcase from #20821

### PR DESCRIPTION
**Pull request by mattcaswell**
An infinite loop was identified via a fuzz testcase in openssl/openssl#20821. We add it to the corpora.

See also openssl/openssl#20824

### CODE REVIEW

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
ping for second review @openssl/committers

**Review by beldmit — APPROVED** [review]
LGTM

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Pull request closed without merging**
